platform/core/src/services/HangingProtocolService/isDisplaySetFromUrl.ts

```diff
@@ -0,0 +1,53 @@
+import { splitComma } from '../../utils';
+
+/** Indicates if the given display set is the one specified in the
+ * displaySet parameter in the URL
+ * The parameters are:
+ *    displaySet.SeriesInstanceUID
+ *    displaySet.SOPInstanceUID
+ */
+const isDisplaySetFromUrl = (displaySet): boolean => {
+  const params = new URLSearchParams(window.location.search);
+  const displaySetSeriesInstanceUID = splitComma(
+    params.getAll('displaySet.SeriesInstanceUID')
+  );
+  let ret;
```

Comment:
please carry over the discussions from the previous PRs. Probably this is an old code though, `ret` is not clear